adding subcommittee directory for CoC/policy group

k8hertweck opened the pull request:
The README contains basic information about the subcommittee, but could be improved with contact information and updating the links at the bottom. 

We should also think about what (if any) documentation to include here, given the nature of the subcommittee. Perhaps just discussion on updates to policies?

wking commented on subcommittees/README.md:
```diff
@@ -21,6 +21,7 @@ task).
 * [Finance](finance/README.md)
 * [Lesson Organization and Develpment](lessons/README.md)
 * [Partner Relations and Fundraising](partner-relations/README.md)
+* [Code of Conduct and Policy](policy-CoC/README.md)
```
I think we should have a consistent name.  Currently you have “Code of Conduct and Policy” here and “Code of Conduct Policy Subcommittee” in the subcommittee's README.  I'd recommend the name be just “Code of Conduct Subcommittee” with a slug of just `CoC`.  Once you've scoped it to “Code of Conduct” I don't think we need to further scope it to “policy” (especially since the README also tasks the subcommittee with enforcement).  And I don't think you want to scope it to “policy” without restricting it to “Code of Conduct” (there are lots of Carpentry policies which do not fall under this subcommittee).

k8hertweck replied:
I used both CoC and policy because this is what the committee itself applied in the formation of the policy. The email is also references policy. Again, not sure why, but I'd prefer to leave it to disambiguate, although I agree this muddies the water of what "policy" means.

ErinBecker replied:
The name of this subcommittee is "Policy Subcommittee". Currently, this group only adjudicates issues related to the CoC but it is written into the description of the committee that they may also be involved with developing and adjudicating other conduct related policies. We decided to leave the name broad to enable this expansion.

wking commented on subcommittees/policy-CoC/README.md:
```diff
@@ -0,0 +1,33 @@
+
+## Purpose and Scope
+
+Enforce the Code of Conduct (CoC) in all parts of the Carpentries community
```
Link to the canonical source(s) for the Code of Conduct and related documents?  Those are written under the guidance of this subcommittee, so it should be easy for folks reading about this subcommittee to find them.  This will also be helpful for future subcommittee members (e.g. so they know where to request updates if the subcommittee's recommendations change).

k8hertweck replied:
I included a link to the CoC, which includes links to the reporting and enforcement guidelines.

wking commented on subcommittees/policy-CoC/README.md:
```diff
@@ -0,0 +1,33 @@
+## Contact
+
+* policy@carpentries.org
+* confidential@carpentries.org
```
A few words (or links) for each of these would be good, to help folks interested in contacting the subcommittee figure out which one to use for their situation.

k8hertweck replied:
I included language to specify the first is to contact the committee, the second to report

wking commented on subcommittees/policy-CoC/README.md:
```diff
@@ -0,0 +1,33 @@
+* Chris Hamm (community)
+* Simon Waldman (community)
+* Erin Becker (staff)
+* Jonah Duckles (staff)
```
Is staff-ness important?  I'm not sure.  If it is, probably add a few words explaining how.

k8hertweck replied:
I'm leaving this for now, as it's a special case among the committees

ErinBecker replied:
@k8hertweck Maybe worth adding the the confidential@ email goes to C. MacDonnell (not a member of the committee). This independent reporting avenue has been used in the past and is a way for people to report issues to the Policy committee without needing to be identified to that group.

k8hertweck replied:
Great point, I've tried make that clearer. If the committee would like to revise this at any point, you're also welcome. I'm trying now to get the basic framework documented!

ErinBecker replied:
Thanks @k8hertweck. I think once the name of the committee is corrected this is ready to merge from my perspective.

wking commented on subcommittees/policy-CoC/README.md:
```diff
@@ -0,0 +1,33 @@
+[blog]: https://software-carpentry.org/blog/
+[blog-archives]: https://software-carpentry.org/blog/categories/#SLUG
+[mailing-list]: http://lists.software-carpentry.org/listinfo/SLUG
+[mailing-list-archives]: http://lists.software-carpentry.org/pipermail/SLUG/
```
You've removed the text which consumes these references so you should either:

* Drop the references.
* Add/restore some text consuming them.

I'd like to see blog links back, or some other words about how the goal of “communicate with the Carpentries community about the CoC” will be carried out.  If you're a member of the Carpentries community interested in receiving that communication, where do you look?

k8hertweck replied:
I tried to clarify this a bit, but am not on the committee so not sure what their intentions are.

k8hertweck commented:
I've addressed comments above and would like to merge. I don't believe the subcommittee will actually be posted minutes (and a comment is added to the subcommittee description to highlight that point), but am leaving the directory there anyway (in case there is discussion later about changes to the policy, or members want to upload documents relating to the development of the enforcement/reporting guidelines).

Tagging @ErinBecker and @jduckles as members of the committee to get an OK to merge

jduckles commented:
Ok by me.
